include/realizations/catchment/Formulation_Manager.hpp

```diff
@@ -22,6 +26,8 @@
 #include "realizations/config/config.hpp"
 #include "realizations/config/layer.hpp"
 
+#include<iostream>
```

Comment:
One other thing I almost missed, is this needed? I didn't see anything specifically that required `<iostream>`

Reply:
Nevermind, I didn't realize this was `Formulation_Manager.hpp` and I know there's at least `std::cerr` outputs in here, so this is fine (especially since it probably includes it transitively anyways)

Reply:
<iostream> is not needed. It was used dusing testing.